From f77cf7151570200d23f30188d07a7b362c714092 Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Tue, 9 Jul 2019 17:42:29 +0530
Subject: [PATCH 0229/1921] octeontx2-pf: Update hardware workarounds for 95xx
 A1 silicon

This patch applies workarounds of 95xx A0 to 95xx A1 silicon

Change-Id: I05a1ee927c274fe8f1bc109fde82d7db3ffb6c53
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12322
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.h b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.h
index 7075f855ee04..15bd50d17fd7 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.h
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.h
@@ -215,7 +215,7 @@ static inline bool is_96xx_A0(struct pci_dev *pdev)
 
 static inline bool is_95xx_A0(struct pci_dev *pdev)
 {
-	return (pdev->revision == 0x10) &&
+	return ((pdev->revision == 0x10) || (pdev->revision == 0x11)) &&
 		(pdev->subsystem_device == PCI_SUBSYS_DEVID_95XX_RVU_PFVF);
 }
 
-- 
2.31.1

